modules/psap-driver-toolkit-using.adoc

```diff
@@ -80,7 +77,6 @@ spec:
       WORKDIR simple-kmod
 
       # Prep and build the module
```

Comment:
The comment should change too, because there is no "prep" any more. Maybe we should say "Build and install the module".

Reply:
`make all` is just building. See https://github.com/openshift-psap/simple-kmod/blob/e8528529b810be4e856e741ea2d2f51ea363795a/Makefile#L12-L14

Reply:
The comment applies for the 2 lines: `make all` and `make install`. It can say "build and install ..."

Reply:
You are right. I have missed the last line.